.github/workflows/post-apiview.yml

```diff
@@ -13,10 +13,10 @@ jobs:
     name: After APIView
     runs-on: ubuntu-24.04
     if: |
-      github.event.check_run.check_suite.app.name == 'Azure Pipelines' && (
+      github.event.check_run.check_suite.app.name == 'Azure Pipelines' && 
+      endsWith(github.event.check_run.external_id, '29ec6040-b234-4e31-b139-33dc4287b756') && (
```

Comment:
It doesn't need to filter the ado instance id here since API view step is also needed for private spec repo.